### ISSUE TITLE
Status Page Resource missing `widget_type` attribute

### ISSUE THREAD

**Issue by theaccordance** [BetterStackHQ/terraform-provider-better-uptime#32]
## User Story

As an owner using terraform to configure BetterUptime, I need to be able to specify the widget type, so Terraform doesn't arbitrarily decide what widget_type to use for me.  

## Background
* While on a sales call recently, it was disclosed that I could use Terraform to configure BetterUptime.  I said _cool, we use terraform at my company and I've been looking for an excuse to learn_, so I went about implementing our BU instance using Terraform 
* As I iterated, I discovered that the provider would update resources I hadn't changed.  While all the values I did specify stayed the same, it would modify the setting for the `widget_type` attribute, changing my page resources from `history` to `plain`.  
* I went to the documentation to identify the property I was missing in my config but there was nothing described that corresponded to the `widget_type`.  I had to learn about `widget_type` through the API docs.  
* Because of this bug, every time I add a new resource to my status page, I have to follow-up in the Web UI and manually set the `widget_type` value for each individual resource to maintain what I want displayed.  Since this is an internal-use status page, it currently has 30 resources listed, which is a lot of manual clicking when you factor in having to expand UI components to even see the dropdown to change the value.

**Comment by theaccordance** [comment]
I'll also add that I gave it the _ol' college try_ to add the attribute to my configs to see if it had undocumented support and as far as I could tell, that was not an option.

**Comment by theaccordance** [comment]
I just realized this morning that a sample of the config may be beneficial: 
```
resource "betteruptime_status_page_resource" "internal_prod_convo_survey" {
  status_page_id         = betteruptime_status_page.internal.id
  status_page_section_id = betteruptime_status_page_section.internal_prod.id
  resource_id            = betteruptime_monitor.pulse_convo_prod.id
  resource_type          = "Monitor"
  public_name            = "Pulse Anonymous Conversations"
  position               = 4
}
```

terraform block: 
```
terraform {
  required_version = "~> 1.0.3"
  required_providers {
    betteruptime = {
      source  = "BetterStackHQ/better-uptime"
      version = "0.3.12"
    }
  }
... 
  ```

Please let me know if you require other samples, I would include the whole file but I want to avoid disclosing my infrastructure details publicly.

**Comment by gyfis** [comment]
Hi @theaccordance ! Thanks a lot for raising the issue, and apologies for the delay of the response.

I checked the code and we do support `widget_type` in the resources (added in this PR: https://github.com/BetterStackHQ/terraform-provider-better-uptime/pull/27 ). Can you try adding e.g. `widget_type = "response_times"` to your terraform resource definition and see if that helps?

**Comment by theaccordance** [comment]
@gyfis Thanks for getting back to me on my issue 😀

I'm pretty sure I did try that, but I'll do it again and capture my terminal output if it doesn't work.  I'll try to get to this in the next week so we can close this issue out with confidence, would try today but we're currently dealing with an outage and I'm not about to disrupt that status page 😅

**Comment by gyfis** [comment]
@theaccordance Thanks a lot, and good luck with the outage!

**Comment by gyfis** [comment]
Hey Joe (@theaccordance)! Did you have a chance to try this again? :) happy to help!

**Comment by theaccordance** [comment]
Hey @gyfis,

Apologies for a delayed response, I was OOO for surgery when you last replied.  

I have yet to try it but I'll do so today so we can either progress on the next troubleshooting step or close the ticket :)

**Comment by theaccordance** [comment]
Update:  I have my test scaffolded with your change, but I hit a blocker trying to test it on my end 😅 

It's probably related to some refactoring my team did while I was OOO to migrate us to Terraform cloud.  I'll get unblocked on Monday and run my test and let you know how it goes.

**Comment by gyfis** [comment]
@theaccordance Hey Joe, how's it going? Did you have a chance to look into the `widget_type` attribute? We released one more version (v0.3.14), but `widget_type` should be accessible in the previous one as well. Let me know how it goes, and we can close the issue afterwards! 🙌

**Comment by gyfis** [comment]
I'm going to close the issue for now, but happy to reopen if the problem gets confirmed!
